index.js

```diff
@@ -253,11 +253,13 @@ Client.prototype._connect = function (stream_id, info) {
  * @api public
  */
 Client.prototype.add = function (info) {
-    var stream, stream_id, streams = [];
+    var stream_id, streams = [];
     var _this = this;
     if (info instanceof Array) {
         // We've been passed multiple server information
         info.forEach(function(network) {
+            var stream = [];
```

Comment:
this doesn't make any difference as `var` declarations get lifted to the top of the function either way